desktop/bin/after_sign_hook.js

```diff
@@ -38,6 +38,7 @@ module.exports = async function ( context ) {
 		appleId: NOTARIZATION_ID,
 		appleIdPassword: NOTARIZATION_PWD,
 		ascProvider: NOTARIZATION_ASC_PROVIDER,
+		teamId: NOTARIZATION_ASC_PROVIDER,
```

Comment:
With this change, notarization succeeds on my machine

<img width="1279" alt="image" src="https://github.com/user-attachments/assets/ec6a3560-637a-4fc1-a67d-feb7189e6960">

However, it might not be enough to work in CI because of a possible lack of code signing identity there (required upstream of notarization)

@wojtekn what is the best way for me to test this? Is it okay to spam the beta channel with new betas, or should I hack the script and CI to attempt the Mac build on every commit without publishing? My preference would be for the latter, to avoid release noise.

Reply:
@mokagio It's okay to spam betas until we get this fixed. The build process won't publish a new beta in https://github.com/Automattic/wp-desktop/releases if it doesn't reach `wp-desktop-publish` step.

> However, it might not be enough to work in CI because of a possible lack of code signing identity there (required upstream of notarization)

As far as I understand the error message, on CI notarize fails because the code is not signed.